.github/workflows/publish.yml

```diff
@@ -31,5 +31,4 @@ jobs:
       - name: Publish to JFrog Artifactory
         run: ./gradlew artifactoryPublish
         env:
-          ARTIFACTORY_USER: ${{ secrets.ARTIFACTORY_USER }}
-          ARTIFACTORY_KEY: ${{ secrets.BESU_ARTIFACTORY }}
\ No newline at end of file
+          ARTIFACTORY_USER: ${{ secrets.BESU_ARTIFACTORY }}
```

Comment:
so you don't need the key? just the user

Reply:
I copied it from Besu workflow files. This can only be tested once pre-release is created and publish is attempted.